cloud-sql/postgres/sqlalchemy/connection_test.py

```diff
@@ -0,0 +1,100 @@
+from contextlib import contextmanager
+import logging
+import os
+from typing import Dict
+import uuid
+
+import pg8000
+import pytest
+
+import main
+
+
+logger = logging.getLogger()
+
+
+@pytest.mark.usefixtures("tcp_db_connection")
+def test_tcp_connection(tcp_db_connection):
+    assert tcp_db_connection is not None
+
+
+@pytest.mark.usefixtures("unix_db_connection")
+def test_unix_connection(unix_db_connection):
+    assert unix_db_connection is not None
+
+
+@pytest.mark.usefixtures('tcp_db_connection')
+def test_get(tcp_db_connection):
+    main.create_tables()
+    context = main.get_index_context()
+    assert isinstance(context, dict)
+    assert len(context.get('recent_votes')) >= 0
+    assert context.get('tab_count') >= 0
+    assert context.get('space_count') >= 0
+
+
+env_map = {
+    'POSTGRES_USER': 'DB_USER',
+    'POSTGRES_PASSWORD': 'DB_PASS',
+    'POSTGRES_DATABASE': 'DB_NAME',
+    'POSTGRES_INSTANCE': 'CLOUD_SQL_CONNECTION_NAME',
+}
+
+
+@pytest.fixture
```

Comment:
nit: should we consider scoping this to `module`? not sure we need a different connection pool for every test